Cronos/cronos-evm-starter/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "cronos-evm-starter",
+  "version": "1.0.0",
+  "description": "A basic Ethermint EVM example project with an event and call handler. Read more about this at https://academy.subquery.network/build/cosmos-evm.html. This project can be use as a starting point for developing your SubQuery project",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "subql build",
+    "prepack": "rm -rf dist && npm build",
+    "test": "jest",
+    "codegen": "subql codegen",
+    "start:docker": "docker-compose pull && docker-compose up --remove-orphans"
+  },
+  "homepage": "https://github.com/subquery/cosmos-subql-starter",
+  "repository": "github:subquery/cosmos-subql-starter",
+  "files": [
+    "dist",
+    "schema.graphql",
+    "project.yaml"
+  ],
+  "author": "SubQuery Team",
+  "license": "MIT",
+  "devDependencies": {
+    "@cosmjs/stargate": "^0.28.9",
+    "@subql/cli": "^1.1.0",
+    "typescript": "^4.7.4"
+  },
+  "dependencies": {
+    "@subql/ethermint-evm-processor": "^0.1.2",
+    "@subql/types-cosmos": "^0.1.0",
+    "@types/node": "^17.0.21",
+    "pino": "^7.8.0",
+    "ts-proto": "^1.112.1",
+    "tslib": "^2.3.1"
+  },
+  "exports": {
+    "ethermintEVM": "./node_modules/@subql/ethermint-evm-processor/dist/index.js"
+  }
```

Comment:
```suggestion
  }
```

Reply:
@stwiname is this no longer needed too?
![image](https://user-images.githubusercontent.com/46693720/192079131-60768deb-4723-4c66-8b42-ec72089a168d.png)
https://academy.subquery.network/build/substrate-evm.html